src/domain/LottoGame.js

```diff
@@ -0,0 +1,115 @@
+import { Lotto } from "./Lotto.js";
+
+export class LottoGame {
+  static LOTTO_PRICE = 1_000;
+  static LOTTO_RANK = {
+    FIRST: {
+      NAME: "first",
+      PRIZE: 2_000_000_000,
+      COUNT: 6,
+    },
+    SECOND: {
+      NAME: "second",
+      PRIZE: 30_000_000,
+      COUNT: 5,
+    },
+    THIRD: {
+      NAME: "third",
+      PRIZE: 1_500_000,
+      COUNT: 5,
+    },
+    FOURTH: {
+      NAME: "fourth",
+      PRIZE: 50_000,
+      COUNT: 4,
+    },
+    FIFTH: {
+      NAME: "fifth",
+      PRIZE: 5_000,
+      COUNT: 3,
+    },
+  };
```

Comment:
객체를 조금 더 응용한다고 가정해본다면,

```suggestion
  static LOTTO_RANK = {
    FIRST: LottoRank.of("first", 2_000_000_000, 6),
    SECOND: LottoRank.of("second", 30_000_000, 5),
    THIRD: LottoRank.of("third", 1_500_000, 5),
    FOURTH: LottoRank.of("fourth", 50_000, 4),
    FIFTH: LottoRank.of("fifth", 5_000, 3),
  };
```

이렇게 만들어서 사용하는 모습을 상상해보시면 좋답니다!